src/app/preferences/general-preferences/general-preferences.service.ts

```diff
@@ -5,7 +5,7 @@ namespace Core {
       'ngInject';
     }
     getDefaultVerticalNavState(): string {
-      if (this.$window.localStorage.getItem('defaultVerticalNavState') !== null) {
+      if (this.$window.localStorage.getItem('defaultVerticalNavState') != undefined) {
```

Comment:
You can use `_.isNil(...)` here.
https://lodash.com/docs/4.17.15#isNull

This is also a good resource when you are considering to use lodash:
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore